.github/workflows/pull-request-rfc.yml

```diff
@@ -0,0 +1,98 @@
+name: RFC Pull Request
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const RFC_CAT_ID = 'DIC_kwDOD1jqbs4CQg5P'
```

Comment:
Can this be moved into a GitHub env secret?

Reply:
It's not a secret, it can be retrieved from L86 to reference to the category `RFC Discussion`. Hard-coded here as I want to avoid doing the query for every single PR.